modules/402-ingress-nginx/hooks/minimal_ingress_nginx_version.go

```diff
@@ -46,6 +46,7 @@ const (
 	minVersionValuesKey     = "ingressNginx:minimalControllerVersion"
 	incompatibleVersionsKey = "ingressNginx:hasIncompatibleIngressClass"
 	disruptionKey           = "ingressNginx:hasDisruption"
+	moduleConfigVersionKey  = "ingressNginx:moduleConfigVersion"
```

Comment:
```suggestion
	configuredDefaultVersionKey  = "ingressNginx:configuredDefaultVersion"
```
moduleConfigVersion is version of ModuleConfig ("1" in our case).